app/models/framework/definition/RM1031.rb

```diff
@@ -9,23 +9,23 @@ class RM1031 < Base
       class Invoice < Sheet
         total_value_field 'Total Charge (Ex VAT)'
 
-        field 'Customer Organisation', :string, exports_to: 'CustomerName'
+        field 'Customer Organisation', :string, exports_to: 'CustomerName', presence: true
         field 'Customer PostCode', :string, exports_to: 'CustomerPostCode'
-        field 'Customer URN', :integer, exports_to: 'CustomerURN'
-        field 'Customer Invoice Date', :date, exports_to: 'InvoiceDate'
-        field 'Customer Invoice Number', :string, exports_to: 'InvoiceNumber'
+        field 'Customer URN', :integer, exports_to: 'CustomerURN', urn: true
+        field 'Customer Invoice Date', :string, exports_to: 'InvoiceDate', ingested_date: true
+        field 'Customer Invoice Number', :string, exports_to: 'InvoiceNumber', presence: true
         field 'Service Type', :string, exports_to: 'ProductGroup'
         field 'Category', :string, exports_to: 'ProductClass'
         field 'Item Code', :string, exports_to: 'ProductCode'
-        field 'UNSPSC', :integer, exports_to: 'UNSPSC'
+        field 'UNSPSC', :string, exports_to: 'UNSPSC', ingested_numericality: { only_integer: true }, allow_nil: true
         field 'Unit of Purchase', :string, exports_to: 'UnitType'
-        field 'Price per Unit', :decimal, exports_to: 'UnitPrice'
-        field 'Quantity', :decimal, exports_to: 'UnitQuantity'
-        field 'Total Charge (Ex VAT)', :decimal, exports_to: 'InvoiceValue'
-        field 'VAT amount charged', :decimal, exports_to: 'VATCharged'
-        field 'Cost Centre ', :string
```

Comment:
Nice catch

Reply:
Although the MISO CSV contains the space, we trim all fields/headers at the ingest stage.